src/app/pages/observations/observation-detail.component.ts

```diff
@@ -71,10 +71,10 @@ export class ObservationDetailComponent {
   documents: ObservationDocument[] = []; // Sorted by name initially
   documentsToDelete: ObservationDocument[] = []; // Existing document to delete
   documentsToUpload: ObservationDocument[] = []; // New document to upload
-  evidence: ObservationDocument = this.datastoreService.createRecord(ObservationDocument, { type: null });
+  evidence: ObservationDocument = this.datastoreService.createRecord(ObservationDocument, {});
   evidenceTypes = [ // Possible types of an evidence
-    'Government document or data', 'Company document or data', 'Photos', 
-    'Maps', 'Testimony from local communities', 'Other'
+    'Government documents', 'Company documents', 'Photos', 
```

Comment:
I checked the [back-end code](https://github.com/Vizzuality/fti_api/blob/develop/app/models/observation.rb#L44-L45) and the types are slightly different:
- “Government documents” is “Government Documents”
- “Company documents” is “Company Documents”

Can you make the change?